charts/opentelemetry-operator/values.yaml

```diff
@@ -6,6 +6,12 @@
 ##
 nameOverride: ""
 
+## Create the opentelemetry-operator-system namespace where the operator will be installed to.
+## Disable this if you have already created the namespace.
+##
+createNamespace:
+  enabled: true
```

Comment:
Why `createNamespace.enabled` instead of just `createNamespace`? the `enabled` part sounds a bit confusing...

Reply:
I think it's just a switch - turn on or turn off. So, I assigned this `enabled`. Maybe it's confusing. Let me remove it.